File: Java/Get_Concatenation.java
class Get_Concatenation {
    public int[] getConcatenation(int[] nums) {
        int size = nums.length*2;
        int[] arr = new int[size];
        System.arraycopy(nums,0,arr,0,nums.length);
        int j = nums.length;
        for (int i = 0; i<nums.length; i++){
            arr[j] = nums[i];
            j++;
        }
        return arr;
    }
}

File: C++/getConcatenation.cpp
#include <iostream>
#include <vector>
using namespace std;

class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> ans(nums.size()*2);
        for (int i = 0; i < nums.size(); i++) {
            ans[i] = nums[i];
        }
        int index = 0;
        for (int j = nums.size(); j < ans.size(); j++) {
            ans[j] = nums[index];
            index++;
        }
        return ans;
    }
};